### DISCUSSION TITLE
cast wallet encryption  error's

### DISCUSSION THREAD

**Discussion by Usengo** [Cyfrin/foundry-full-course-cu#2227] [Q&A]
I constantly  have error when ever i try to use CAST  to encrypt  my private key.
please can anyone give the methodology to set CAST to encrypt private key without error.

ERROR EXAMPLE 

Error: 
Keystore password file `"your_keystore_password"` does not exist

**Comment by EngrPips** [comment]
Hello @Usengo, You should follow the instructions here https://book.getfoundry.sh/reference/cast/cast-wallet-import, and I think you will be successful.
